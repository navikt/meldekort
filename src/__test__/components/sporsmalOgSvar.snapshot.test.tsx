import * as React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ProviderWrapper from '../utils/providerWrapper';
import SporsmalOgSvarVisning from '../../app/components/meldekortdetaljer/sporsmalvisning/sporsmalOgSvar';

Enzyme.configure({ adapter: new Adapter() });

it('SporsmalOgSvarVisning', () => {
  const tree = shallow(
    <ProviderWrapper>
      <SporsmalOgSvarVisning sporsmalOgSvar={sos} />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});

const sos = [
  {
    kategori: 'arbeid',
    sporsmal: 'sporsmal.arbeid',
    forklaring: 'forklaring.sporsmal.arbeid',
    svar: true,
    formatertDato: '20.01.2019',
  },
  {
    kategori: 'aktivitetArbeid',
    sporsmal: 'sporsmal.aktivitetArbeid',
    forklaring: 'forklaring.sporsmal.aktivitetArbeid',
    svar: false,
    formatertDato: '20.01.2019',
  },
];
