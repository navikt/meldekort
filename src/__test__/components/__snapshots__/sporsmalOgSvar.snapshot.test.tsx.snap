// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SporsmalOgSvarVisning 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(IntlProvider)
    defaultLocale="nb"
    locale="nb"
  >
    <SporsmalOgSvarVisning
      sporsmalOgSvar={
        Array [
          Object {
            "forklaring": "forklaring.sporsmal.arbeid",
            "formatertDato": "20.01.2019",
            "kategori": "arbeid",
            "sporsmal": "sporsmal.arbeid",
            "svar": true,
          },
          Object {
            "forklaring": "forklaring.sporsmal.aktivitetArbeid",
            "formatertDato": "20.01.2019",
            "kategori": "aktivitetArbeid",
            "sporsmal": "sporsmal.aktivitetArbeid",
            "svar": false,
          },
        ]
      }
    />
  </Connect(IntlProvider)>
</Provider>
`;
